src/foam/u2/crunch/wizardflow/TestAgent.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * @license
+ * Copyright 2020 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.u2.crunch.wizardflow',
+  name: 'TestAgent',
+
+  implements: [ 'foam.core.ContextAgent' ],
+
+  imports: [
+    'capabilities',
+    'wizardlets'
+  ],
+
+  methods: [
+    // If Property expressions ever unwrap promises this method can be blank.
+    function execute() {
+      console.log('caps', this.capabilities);
```

Comment:
logs

Reply:
`TestAgent` is just for displaying these console logs. I'd like to keep it here but we could comment out TestAgent in the sequence